server/views/partials/layout.njk

```diff
@@ -65,7 +65,7 @@
       {% include "./prisonerProfileBanner.njk" %}
     </aside>
     {% block subNavigation %}
-        {% if prisoner.prisonerNumber !== undefined %}
+        {% if prisoner.prisonerNumber !== undefined and pageTitle === 'Adjustments - Home' %}
```

Comment:
I don't think we should be putting logic like this dependent on the page title.

I'd suggest changing this to {% if prisoner.prisonerNumber !== undefined and !hideSubNav %}

Then within the hub.njk do {% set hideSubNav = true %}

Reply:
I have adjusted this logic